plugins/collections/get.js

```diff
@@ -5,123 +5,6 @@ const joi = require('joi');
-        .then(prJobs => {
-            if (!prJobs) {
-                return prs;
-            }
-
-            // Return array of prJobs' builds
-            return Promise.all(prJobs.map(job => job.getBuilds()));
-        })
-        .then(prJobsBuilds => {
-            if (!prJobsBuilds || !prJobsBuilds.length) {
-                return prs;
-            }
-
-            // prJobsBuilds is a 2D array where each element is an array of builds
-            // for one PR job.
-            prs.open = prJobsBuilds.length;
-
-            prJobsBuilds.forEach(jobBuilds => {
-                // Check the first element (last build) if pr job failed
-                if (Array.isArray(jobBuilds) && typeof jobBuilds[0] === 'object') {
-                    if (jobBuilds[0].status === 'FAILURE') {
-                        prs.failing += 1;
-                    }
-                }
-            });
-
-            return prs;
-        })
-        .catch(err => {
-            logger.error(err);
-
-            return prs;
-        });
-}
-
-/**
- * Helper function to get last builds of pipeline
- *
- * @param   {Object}  pipeline      Pipeline to get last builds for
- * @param   {Factory} eventFactory  Factory for getting last event model
- * @returns {Object}
- */
-function getPipelineHealth(pipeline, eventFactory) {
-    let lastBuilds = [];
-
-    if (!pipeline.lastEventId) {
-        return lastBuilds;
-    }
-
-    return eventFactory
-        .get(pipeline.lastEventId)
-        .then(event => {
-            if (!event) {
-                return lastBuilds;
-            }
-
-            return event.getBuilds().then(builds => {
-                if (builds.length) {
-                    // The events are sorted by most recent first. Need to reverse the order
-                    // to allow for matching with workflow job on the UI
-                    lastBuilds = Promise.all(builds.map(b => b.toJsonWithSteps())).then(bs => bs.reverse());
-                }
-
-                return lastBuilds;
-            });
-        })
-        .catch(err => {
-            logger.error(err);
-
-            return lastBuilds;
-        });
-}
-
-/**
- * Helper function to populate pipelines with last builds and pr info
- *
- * @param   {Array}   pipelines    Array of pipeline models to get last builds for
- * @param   {Factory} eventFactory Factory for getting last event model
- * @returns {Array}
- */
-function getPipelinesInfo(pipelines, eventFactory) {
-    return (
-        pipelines
-            // Filter out pipelines that don't exist
-            .filter(pipeline => !!pipeline)
-            .map(pipeline =>
-                // Get the PR Info and last builds for each pipeline
-                Promise.all([getPipelinePRInfo(pipeline), getPipelineHealth(pipeline, eventFactory)])
-                    // Combine the PR Info and last builds and return the populated pipeline
-                    .then(([pipelinePRInfo, pipelineHealth]) => {
```

Comment:
are we not planning to show any of this data in the collections?

Reply:
They are not used anywhere, this was was before we had build metrics. for the old collections view.